Polyhedron/demo/Polyhedron/CMakeLists.txt

```diff
@@ -17,6 +17,9 @@ if(POLICY CMP0074)
   cmake_policy(SET CMP0074 NEW)
 endif()
 
+if(POLICY CMP0115)
+  cmake_policy(SET CMP0115 OLD)
+endif()
```

Comment:
@lrineau better this way ?

Reply:
The warning related to [`CMP0115`](https://cmake.org/cmake/help/git-stage/policy/CMP0115.html) explains that our CMake code is doing something fishy. Instead of hidden that warning by using the `OLD` behavior (that is fishy, and deprecated by design), the solution is to actually fix the issue. I have stated that in https://github.com/CGAL/cgal/pull/4243/commits/25e65bcedacdad2f5f4645f695cda65da0994b31.